metrics_gocdb.py

```diff
@@ -172,6 +172,72 @@ def get_queries():
     result = elastic.search(index="logstash-" + date, body=params_dict)
     return result["hits"]["total"]
 
+
+def get_unique_ips_over_28_days():
+    """
+    Get the number of unique IP accessing GOCDB over the last 28 days.
+
+    Only accesses via the load balancers will be counted.
+    """
+    date = datetime.now()
+    # Use a set to leverage the fact duplicates are handled by the data
+    # structure.
+    unique_ips = set()
+    # For the last 28 days (not counting today), query the unique IPs per day.
+    for _ in range(0, 28):
+        date = date - timedelta(1)
```

Comment:
Can be more explicit:
```suggestion
        date = date - timedelta(days=1)
```

Reply:
+1